Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSupportCards.java

```diff
@@ -575,496 +577,59 @@ public class ScryfallImageSupportCards {
-            put("WAR/Dovin, Hand of Control/229*", "https://api.scryfall.com/cards/war/229★/");
-            put("WAR/Gideon Blackblade/13*", "https://api.scryfall.com/cards/war/13★/");
-            put("WAR/Huatli, the Sun's Heart/230*", "https://api.scryfall.com/cards/war/230★/");
-            put("WAR/Jace, Wielder of Mysteries/54*", "https://api.scryfall.com/cards/war/54★/");
-            put("WAR/Jaya, Venerated Firemage/135*", "https://api.scryfall.com/cards/war/135★/");
-            put("WAR/Jiang Yanggu, Wildcrafter/164*", "https://api.scryfall.com/cards/war/164★/");
-            put("WAR/Karn, the Great Creator/1*", "https://api.scryfall.com/cards/war/1★/");
-            put("WAR/Kasmina, Enigmatic Mentor/56*", "https://api.scryfall.com/cards/war/56★/");
-            put("WAR/Kaya, Bane of the Dead/231*", "https://api.scryfall.com/cards/war/231★/");
-            put("WAR/Kiora, Behemoth Beckoner/232*", "https://api.scryfall.com/cards/war/232★/");
-            put("WAR/Liliana, Dreadhorde General/97*", "https://api.scryfall.com/cards/war/97★/");
-            put("WAR/Nahiri, Storm of Stone/233*", "https://api.scryfall.com/cards/war/233★/");
-            put("WAR/Narset, Parter of Veils/61*", "https://api.scryfall.com/cards/war/61★/");
-            put("WAR/Nicol Bolas, Dragon-God/207*", "https://api.scryfall.com/cards/war/207★/");
-            put("WAR/Nissa, Who Shakes the World/169*", "https://api.scryfall.com/cards/war/169★/");
-            put("WAR/Ob Nixilis, the Hate-Twisted/100*", "https://api.scryfall.com/cards/war/100★/");
-            put("WAR/Ral, Storm Conduit/211*", "https://api.scryfall.com/cards/war/211★/");
-            put("WAR/Saheeli, Sublime Artificer/234*", "https://api.scryfall.com/cards/war/234★/");
-            put("WAR/Samut, Tyrant Smasher/235*", "https://api.scryfall.com/cards/war/235★/");
-            put("WAR/Sarkhan the Masterless/143*", "https://api.scryfall.com/cards/war/143★/");
-            put("WAR/Sorin, Vengeful Bloodlord/217*", "https://api.scryfall.com/cards/war/217★/");
-            put("WAR/Tamiyo, Collector of Tales/220*", "https://api.scryfall.com/cards/war/220★/");
-            put("WAR/Teferi, Time Raveler/221*", "https://api.scryfall.com/cards/war/221★/");
-            put("WAR/Teyo, the Shieldmage/32*", "https://api.scryfall.com/cards/war/32★/");
-            put("WAR/The Wanderer/37*", "https://api.scryfall.com/cards/war/37★/");
-            put("WAR/Tibalt, Rakish Instigator/146*", "https://api.scryfall.com/cards/war/146★/");
-            put("WAR/Ugin, the Ineffable/2*", "https://api.scryfall.com/cards/war/2★/");
-            put("WAR/Vivien, Champion of the Wilds/180*", "https://api.scryfall.com/cards/war/180★/");
-            put("WAR/Vraska, Swarm's Eminence/236*", "https://api.scryfall.com/cards/war/236★/");
             // SLD
             // fake double faced cards
+            put("SLD/Ajani Goldmane/1453b", "https://api.scryfall.com/cards/sld/1453/en?format=image&face=back");
```

Comment:
curious what's going on with the reordering here? (not necessarily a problem)

Reply:
Alphabetical so that specific cards were easier to find.